materialize-snowflake/snowflake.go

```diff
@@ -399,33 +415,27 @@ func (d *transactor) Commit(ctx context.Context) error {
 	}
 	defer txn.Rollback()
 
-	// Apply the client's prepared checkpoint to our fence.
-	if err = d.store.fence.Update(ctx,
-		func(ctx context.Context, sql string, arguments ...interface{}) (rowsAffected int64, _ error) {
-			if result, err := txn.ExecContext(ctx, sql, arguments...); err != nil {
-				return 0, fmt.Errorf("txn.Exec: %w", err)
-			} else if rowsAffected, err = result.RowsAffected(); err != nil {
-				return 0, fmt.Errorf("result.RowsAffected: %w", err)
-			}
-			return
-		},
-	); err != nil && strings.Contains(err.Error(), "timeout") {
```

Comment:
I'm curious about this "timeout" checking. It seems bad to check for a string within an error string, so I feel like it is good to get rid of this. Did it provide any useful information at all?

Reply:
Reading the error message, I think the idea here is to give recommendations to the user for resolving the issue, by increasing `LOCK_TIMEOUT` and `STATEMENT_TIMEOUT_IN_SECONDS`, so perhaps it is actually useful. I can bring it back 🤔